bootstrap_apollo.sh

```diff
@@ -3,7 +3,7 @@
 export GALAXY_SHARED_DIR=`pwd`/apollo_shared_dir
 mkdir -p "$GALAXY_SHARED_DIR"
 
-docker run -d -it -p 8888:8080 -v `pwd`/apollo_shared_dir/:`pwd`/apollo_shared_dir/ quay.io/gmod/apollo:2.4.1
+docker run --memory=4g -d -it -p 8888:8080 -v `pwd`/apollo_shared_dir/:`pwd`/apollo_shared_dir/ quay.io/gmod/apollo:latest
```

Comment:
I don't think the --memory arg is needed, by default it's unlimited

Reply:
https://docs.docker.com/v17.12/docker-for-mac/

Its 2 GB by default on the mac, so it is super-sluggish unless I set this.